buildSrc/src/main/java/org/elasticsearch/gradle/tar/SymoblicLinkPreservingTarPlugin.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.gradle.tar;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+
+public class SymoblicLinkPreservingTarPlugin implements Plugin<Project> {
```

Comment:
Adding a nop plug-in should not be necessary,  the task will be accessible as long as it's on the classpath.

Reply:
Without this, in my tests, I ran into:

```
* What went wrong:
Could not compile build file '/Users/jason/src/elastic/elasticsearch/buildSrc/src/testKit/symbolic-link-preserving-tar/build.gradle'.
> startup failed:
  build file '/Users/jason/src/elastic/elasticsearch/buildSrc/src/testKit/symbolic-link-preserving-tar/build.gradle': 1: unable to resolve class org.elasticsearch.gradle.tar.SymbolicLinkPreservingTar
   @ line 1, column 1.
     import org.elasticsearch.gradle.tar.SymbolicLinkPreservingTar
     ^
  
  1 error
```

Reply:
ok, Looks like the testkit class path magic only works for plugins